docs/pages/build-reference/e2e-tests.mdx

```diff
@@ -135,54 +135,73 @@ Let's add two development dependencies to the project - `jest` and `detox`. `jes
 
 Detox requires you to specify both the build command and path to the binary produced by it. Technically, the build command is not necessary when running tests on EAS Build, but allows you to run tests locally (for example, using `npx detox build --configuration ios.release`).
 
-Edit **.detoxrc.json** and replace the configuration with:
+Edit **detox.config.js** and replace the configuration with:
 
-```json .detoxrc.json
-{
-  "testRunner": "jest",
-  "runnerConfig": "e2e/config.json",
-  "skipLegacyWorkersInjection": true,
-  "apps": {
-    "ios.release": {
-      "type": "ios.app",
+```js detox.config.js
+/** @type {Detox.DetoxConfig} */
+module.exports = {
+  logger: {
+    level: process.env.CI ? 'debug' : undefined,
+  },
+  testRunner: {
+    $0: 'jest',
+    args: {
+      config: 'e2e/jest.config.js',
+      _: ['e2e'],
+    },
+  },
+  artifacts: {
+    plugins: {
+      log: process.env.CI ? 'failing' : undefined,
+      /* @info In Detox 20, this plugin setting will take screenshots if tests fail */
+      screenshot: 'failing',
+      /* @end */
+    },
+  },
+  apps: {
+    'ios.release': {
+      type: 'ios.app',
       /* @info This is project-specific, replace eastestsexample with correct value */
-      "build": "xcodebuild -workspace ios/eastestsexample.xcworkspace -scheme eastestsexample -configuration Release -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build",
-      "binaryPath": "ios/build/Build/Products/Release-iphonesimulator/eastestsexample.app"
+      build:
+        'xcodebuild -workspace ios/eastestsexample.xcworkspace -scheme eastestsexample -configuration Release -sdk iphonesimulator -arch x86_64 -derivedDataPath ios/build',
+      binaryPath:
+        'ios/build/Build/Products/Release-iphonesimulator/eastestsexample.app',
       /* @end */
     },
-    "android.release": {
-      "type": "android.apk",
+    'android.release': {
+      type: 'android.apk',
       /* @info This is project-specific, replace eastestsexample with correct value */
```

Comment:
For Android, I don't see `eastestsexample` in this snippet. Is there is some other value that needs replacement?

Reply:
Good catch, you are correct

Reply:
Fixed